### DISCUSSION TITLE
FFXIV Specific Features

### DISCUSSION THREAD

**Discussion by ArcaneDisgea** [the-convocation/venat#79] [Ideas]
A list of features for the ffxiv modules of the bot.
- [ ] XIV
	- [ ] Lodestone Data #73 
		- [ ] Characters
			- [ ] Character Attributes
			- [ ] Character Classes
			- [ ] Character Crafters
			- [ ] Character Saving
			- [ ] Character Achievements 
            - [ ] Character Cards/Signatures Generation
                - [ ] Equipment
                - [ ] Glamour
                - [ ] Attributes
                - [ ] Standard Jobs/Chara info
                - [ ] Emulate Lodestone cards?
                - [ ] Adventurer Plates ;)
		- [ ] FC Information
            - [ ] FC Members
                - [ ] Ranks
                - [ ] Lodestone Profiles
                - [ ] Names
            - [ ] World/Global Rankings
                - [ ] Weekly
                - [ ] Monthly
		- [ ] PVP Stats
            - [ ] Teams & Members (see fc for types of details)
		- [ ] Deep Dungeon Solo/Group Rankings
            - [ ] Weekly
            - [ ] Monthly
		- [ ] Server Status
		- [ ] Maintenance
	- [ ] Item Queries #74 
	- [ ] Action Queries #74 
	- [ ] Recipe Queries #74 
	- [ ] FFLogs #76 
		- [ ] Top 5 of X Fight
		- [ ] Specific Characters Best logs of current tier.
	- [ ] Monthly/Weekly/Daily Resets #78 
		- [ ] Fashion Report
		- [ ] Custom Deliveries?
		- [ ] Festivals

**Comment by MKhayle** [comment]
We can close this now that I split it in several issues. Once again, thanks arc for setting this up in the first place
